src/shared/loop-util.c

```diff
@@ -182,7 +194,8 @@ static int loop_configure_fallback(int fd, const struct loop_config *c) {
 
         for (unsigned n_attempts = 0;;) {
                 if (ioctl(fd, LOOP_SET_STATUS64, &info_copy) >= 0)
-                        break;
+                        if (ioctl(fd, LOOP_SET_BLOCK_SIZE, c->block_size) >= 0)
+                                break;
```

Comment:
i think if we fail to set the block size this should be something to log about, but not really affect our codeflow otherwise. if we can't set the block size for some reason, we should  debug log, but just continue